README.md

```diff
@@ -46,29 +48,39 @@ In an ideal world, all websites are designed and developed so everyone can use t
-    Luca Pircher
-  </a>
-  ,
-  <a href="https://github.com/thomasdax98" target="_blank" rel="noopener noreferrer nofollow">
-    Thomas Dax
-  </a>
-  and
-  <a href="https://github.com/hntrhfr" target="_blank" rel="noopener noreferrer nofollow">
-    Michael Hinterhofer</a>.
-  <span className="font-medium">Interface and Corporate Design</span> by
-  <a href="https://johannawicht.com/" target="_blank" rel="noopener noreferrer nofollow">
-    Johanna Wicht
-  </a>
-  and
-  <a href="https://www.fabianhellerdesign.com/" target="_blank" rel="noopener noreferrer nofollow">
-    Fabian Heller</a>.
-</p>
+<span aria-label="Allyphant">a11yphant</span> started as a master’s project created by six students at the <a href="https://www.fh-salzburg.ac.at/en/" target="_blank" rel="noopener noreferrer nofollow">
+Salzburg University of Applied Sciences</a> in Austria. The interdisciplinary team consisted of four web engineering and two communication design students.
+
+Nowadays, the a11yphant core-team continues developing a11yphant as Open Source project. Their goal is to make the web a more accessible and inclusive place.
+
+### Concept and Development
+
+<img src="https://unavatar.io/github/dnikub" height="138">
+
+<a href="https://dnikub.dev"  target="_blank" rel="noopener noreferrer nofollow">Daniela Kubesch</a> is a frontend developer who is passionate about accessibility and inclusive design. She strongly believes in equality and inclusion. Daniela is co-founder of a11yphant. When she is not coding, she juggles all things of organisational nature.
+
+<img src="https://unavatar.io/github/lucapircher" height="138">
+
+<a href="https://lucapircher.at" target="_blank" rel="noopener noreferrer nofollow">Luca Pircher</a> is a web developer based in Austria who firmly believes the web should be inclusive. At a11yphant, Luca supports the team as the "Technical Project Lead" by ensuring that the project is a joy to work on for the dev team and planning the technical requirements for upcoming challenges.
+
+<img src="https://unavatar.io/github/thomasdax98" height="138">
+
+<a href="https://github.com/thomasdax98" target="_blank" rel="noopener noreferrer nofollow">Thomas Dax</a> is a full-stack developer interested in product development and developer experience. As "Head of Text", he has supported the team with copywriting since co-founding a11yphant. Thomas further does a lot of frontend development work and takes care of the development setup.
+
+<img src="https://unavatar.io/github/hntrhfr" height="138">
+
+<a href="https://github.com/hntrhfr" target="_blank" rel="noopener noreferrer nofollow">Michael Hinterhofer</a> is a backend developer at <a href="https://www.chargepoint.com/de-de"  target="_blank" rel="noopener noreferrer nofollow">Chargepoint</a> on weekdays and chief of happiness officer for a11yphant on the weekend. Since co-founding a11yphant, he supported the team with his magic in the backend.
```

Comment:
```suggestion
<a href="https://github.com/hntrhfr" target="_blank" rel="noopener noreferrer nofollow">Michael Hinterhofer</a> is a corporate backend developer on weekdays and chief of happiness officer for a11yphant on the weekend. Since co-founding a11yphant, he supported the team with his magic in the backend.
```